### ISSUE TITLE
Home -> SiteMapFooter Missing Headings 

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#773]
Three lists of links are present in this component.  They are each preceded by a title which is visually styled to look like a heading (“Resources”, “Connect” and “Info”).  These titles are only marked up as < div> elements, meaning that their semantic meaning and structure are not conveyed to screen reader users.  It also means that a user cannot easily navigate between them. 
 
Suggested Solution 
 
Mark up each < div> with a className attribute of “{ footer.listTitle}” as a heading.  The headings should be at level 3 because the overall footer should always begin with a heading at level 2 (see “Problematic Heading Placement” in the “MainFooter” component section).  Be sure to use native < h3> elements, rather than adding ARIA attributes to simply reclassify the existing < div> containers. 
 
These headings should then be used to provide an accessible label for the corresponding list, as follows: 
 
• To each < h3>, add a unique DOM ID (id attribute). 
• Use this ID as the value of an aria-labelledby attribute on the opening < ul> tag of the list which follows the heading.

**Comment by johnhbenetech** [comment]
Proposed Code Sample 
 
File: src/components/Home/components/SiteMapFooter/index.js 
 
Note: only one list is shown, but the implementation for all three should follow the same pattern

`<h3 id="listTitle__resources" className={footer.listTitle}>{Locales.strings.resources}</h3> <ul aria-labelledby="listTitle__resources"> `

**Comment by jscholes** [comment]
@rupeshparab - looking good.  The `<ul>` following the `"listTitle__connect"` heading needs an accessible name, and then this can be closed.

```
<h3 id="listTitle__connect" className={footer.listTitle}>{Locales.strings.connect}</h3>
<ul aria-labelledby="listTitle__connect">
```

**Comment by rupeshparab** [comment]
I have added the heading to list

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/865b1b79a31d1117ee0d1ef252924aa715a7ba57.
